Mathlib/Analysis/SpecialFunctions/Trigonometric/Arctan.lean

```diff
@@ -200,6 +200,83 @@ theorem arccos_eq_arctan {x : ℝ} (h : 0 < x) : arccos x = arctan (sqrt (1 - x
   · linarith only [arcsin_pos.2 h]
 #align real.arccos_eq_arctan Real.arccos_eq_arctan
 
+theorem arctan_inv_of_pos {x : ℝ} (h : 0 < x) : arctan x⁻¹ = π / 2 - arctan x := by
+  rw [← arctan_tan (x := _ - _), tan_pi_div_two_sub, tan_arctan]
+  · norm_num; exact (arctan_lt_pi_div_two x).trans (half_lt_self_iff.mpr pi_pos)
+  · rw [sub_lt_self_iff, ← arctan_zero]; exact tanOrderIso.symm.strictMono h
```

Comment:
My reading of the style guide is that putting each tactic call on a single line is slightly preferred. Can you do this (also in the other places)?

Reply:
Existing code in `Arctan.lean` already contains some instances of multiple tactics on the same line (e.g. `arctan_eq_arccos`). I'm keeping my own instances of this in `Arctan.lean` but not in `Basic.lean`.